lib/Db.php

```diff
@@ -39,11 +40,11 @@ public static function reset()
     }
 
     /**
-     * @return Connection
+     * @return ConnectionInterface|Connection
      */
     public static function get()
     {
-        /** @var Connection $db */
+        /** @var ConnectionInterface|Connection $db */
         $db = \Pimcore::getContainer()->get('database_connection');
```

Comment:
```suggestion
        $db = \Pimcore::getContainer()->get('doctrine.dbal.default_connection');
```
We should use `doctrine.dbal.default_connection` here as well, to be consistent, right?